### ISSUE TITLE
The OpenID Connect docs mention a form that doesn't exist on EMS admin panel

### ISSUE THREAD

**Issue by psrpinto** [vector-im/ems-docs#52]
The [OpenID Connect docs](https://ems-docs.element.io/authentication/OpenID-Connect.html#in-the-element-matrix-services-configuration-form) mention a form that doesn't exist on EMS admin panel.

> In the Element Matrix Services configuration form (...)

I'm guessing that form is only available for Enterprise Cloud servers, since SSO is only on that tier. I think this should be mentioned in the OpenID Connect docs.

**Comment by jaller94** [comment]
Yes, all authentication providers are only available for the Element Enterprise Cloud plan.

**Comment by psrpinto** [comment]
Thanks for confirming this @jaller94.

I think it would be useful to have that mentioned in the [OpenID Connect docs](https://ems-docs.element.io/authentication/OpenID-Connect.html#in-the-element-matrix-services-configuration-form), otherwise it's very confusing. For a user who isn't aware that SSO is only available for Enterprise, it either looks like a bug in the EMS admin panel (SSO form is missing) or a bug in the docs (it mentions a non-existing form).
